compass/landice/tests/ismip6_forcing/atmosphere/process_smb.py

```diff
@@ -0,0 +1,411 @@
+import os
+import numpy as np
+import shutil
+import subprocess
+import xarray as xr
+from compass.landice.tests.ismip6_forcing.atmosphere.create_mapfile_smb \
+    import build_mapping_file
+from mpas_tools.io import write_netcdf
+from compass.step import Step
+
+
+class ProcessSMB(Step):
+    """
+    A step for processing the ISMIP6 surface mass balance data
+    """
+
+    def __init__(self, test_case, input_file=None):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.landice.tests.ismip6_forcing.atmosphere.Atmosphere
+            The test case this step belongs to
+
+        input_file : file name of ismip6 forcing data processed by this step
+        """
+        self.input_file = input_file
+        super().__init__(test_case=test_case, name='process_smb', cores=4,
+                         min_cores=1)
+
+    def setup(self):
+        """
+        Set up this step of the test case
+        """
+        config = self.config
+        section = config['ismip6_ais']
+        base_path_ismip6 = section.get('base_path_ismip6')
+        base_path_mali = section.get('base_path_mali')
+        output_base_path = section.get('output_base_path')
+        mali_mesh_file = section.get('mali_mesh_file')
+        period_endyear = section.get("period_endyear")
+        model = section.get("model")
+        scenario = section.get("scenario")
+
+        self.add_input_file(filename=mali_mesh_file,
+                            target=os.path.join(base_path_mali,
+                                                mali_mesh_file))
+
+        input_file_list = self._files[period_endyear][model][scenario]
+        for file in input_file_list:
+            self.add_input_file(filename=os.path.basename(file),
+                                target=os.path.join(base_path_ismip6,
+                                                    file))
+
+        output_file_esm = f"processed_SMB_{model}_{scenario}_" \
+                          f"{period_endyear}.nc"
+        self.add_output_file(filename=output_file_esm)
+
+        # add processed racmo data as input as it is needed for smb correction
+        racmo_clim_file = f"processed_RACMO2.3p2_ANT27" \
+                     f"_smb_climatology_1995-2017.nc"
+        racmo_path = f'{output_base_path}/atmosphere_forcing/' \
+                      f'RACMO_climatology_1995-2017'
```

Comment:
```suggestion
        racmo_clim_file = f"processed_RACMO2.3p2_ANT27" \
                          f"_smb_climatology_1995-2017.nc"
        racmo_path = f"{output_base_path}/atmosphere_forcing/" \
                     f"RACMO_climatology_1995-2017"
```
Make sure indentation matches and try to be consistent about using either single or double quotes everywhere.